ResumeCss.css

```diff
@@ -0,0 +1,339 @@
+
+#First
+{
+	box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+	height:140%;

```

Comment:
do not provide height try to manage it with inner content and also why do you need 140%